SharedPackages/BrowserServicesKit/Sources/PixelKit/PixelKit.swift

```diff
@@ -59,26 +48,42 @@ public final class PixelKit {
         /// This is useful in situations where pixels receive spikes in volume, as the daily pixel can be used to determine how many users are actually affected.
         case dailyAndStandard
 
+        /// [Legacy] Used in Pixel.fire(...) as .unique but without the `_u` requirement in the name
+        case legacyInitial
+
+        /// [Legacy] Used in Pixel.fire(...) as .daily but without the `_d` automatically added to the name
+        case legacyDailyNoSuffix
+
+        /// [Legacy] Sent once per day. The last timestamp for this pixel is stored and compared to the current date. Pixels of this type will have `_d` appended to their name.
+        case legacyDaily
```

Comment:
Summary of changes:
* `legacyDaily` (no suffix) was renamed to `legacyDailyNoSuffix`,
* `daily` (`_d` suffix) was renamed to `legacyDaily`,
* new `daily` was added (`_daily` suffix) <- this is currently only used in PreferencesSidebarModel.